.gitignore

```diff
@@ -0,0 +1,3 @@
+
+.idea
+*/target
```

Comment:
We want to add *.class files right to get right of the java binaries.

Reply:
added